packages/devtools_shared/pubspec.yaml

```diff
@@ -16,4 +16,5 @@ dependencies:
   webkit_inspection_protocol: '>=0.5.0 <2.0.0'
 
 dev_dependencies:
+  dart_code_metrics: ^5.7.2
   test: ^1.21.2
```

Comment:
Can you comment why the dependency is needed?

Reply:
Hmm having it as a dev dependency is part of the installation instructions, but I don't why it's necessary https://dartcodemetrics.dev/docs/getting-started/installation

I  believe @kenzieschmoll was running into an issue due to not having this as a dev dependency. 

I already added it as a dev dependency to devtools_app and devtools_test

Reply:
From the [instructions](https://dartcodemetrics.dev/docs/cli#available-commands), it was necessary to run commands like: `dart run dart_code_metrics:metrics check-unused-code lib`, `check-unused-files`, etc.

Reply:
but maybe the same can be accomplished by using the `dcm` executable directly.. so maybe it isn't needed after all.

Reply:
@incendial this is something that may need docs updates. That you need _either_ the dev dependency _or_ the dcm executable, but not necessarily both.

Reply:
You're referring to the old website, please use the new one instead (https://dcm.dev/docs/teams/getting-started/ and https://dcm.dev/docs/teams/cli/).

There are two versions available: one is a dart package and requires the dependency to be listed and another one is a standalone executable (former is the "Individuals" version and the latter is "Teams). You have a Teams version, so no dependency is needed.

In our docs they are also separated:
<img width="368" alt="Screenshot 2023-04-20 at 11 08 13" src="https://user-images.githubusercontent.com/21116742/233287684-eaa5e252-a3d7-4e1f-bd96-266ef791eb5e.png">
<img width="428" alt="Screenshot 2023-04-20 at 11 08 39" src="https://user-images.githubusercontent.com/21116742/233287800-36bbe034-38a9-40f2-bc17-c8669889ff32.png">

And the url is different.

Hope this helps resolve the misunderstanding.